share/mk/bsd.cpu.mk

```diff
@@ -409,22 +409,40 @@ CFLAGS += ${CFLAGS.${MACHINE_ARCH}}
 CXXFLAGS += ${CXXFLAGS.${MACHINE_ARCH}}
 
 #
-# MACHINE_ABI is a list of properties about the ABI used for MACHINE_ARCH.
+# MACHINE_ABI is a list of properties about default ABI for MACHINE_ARCH.
```

Comment:
I think the old text ("the ABI") here is better?  Each ABI has a distinct MACHINE_ARCH.